data/json/snippets/fliers.json

```diff
@@ -302,6 +302,10 @@
       {
         "id": "flier_75",
         "text": "THE RIPLEY-9k!  Pilot your own cargo exoskeleton now!  Our exoskeletons allow the power of a forklift guided by the strength of your own muscles.  Carry heavy loads over rough terrain, even up stairs!"
+      },
+      {
+        "id": "flier_76",
+        "text": "This is a poster advertising a brand of a hybrid gas-electric vehicle.  Poster claims that hybrid car is the \"best of two worlds\" with the acceleration capabilities of a sport car and fuel consumption of a compact car.  Something called the \"Smart Controller®\" is mentioned several times, which, if you are to believe the poster, it is some kind of military technology adapted for civilian market and it works like magic.  You are not sure if this is something actually useful or just a marketing bullshit."
```

Comment:
This works too much of the player's opinion in. For the most part flier snippets just describe what you read, and the player can imagine what their character thinks of it.